deployment-k8s/src/main/resources/k8s-bash-scripts/copyExampleData.sh

```diff
@@ -45,15 +45,15 @@ rm -rfv "$CLASSPATH"
 echo "Removed previous run classpath-jars for $CLASSPATH"
```

Comment:
Check the mode on this file, it's been changed from 755 to 644 (no execute bit set)